### PR TITLE
Deploy dataplane from openstack-operator

### PR DESCRIPTION
**Pull request by fao89**
Tested at https://github.com/openstack-k8s-operators/openstack-operator/pull/765

### CODE REVIEW

**Comment by fao89** [comment]
recheck

File: Makefile [hunk 1]
```diff
@@ -359,10 +359,6 @@ BMH_NAMESPACE           ?= ${NAMESPACE}
 BAREMETAL_OS_CONTAINER_IMG ?=
 
 # Dataplane Operator
-DATAPLANE_IMG                                    ?= quay.io/openstack-k8s-operators/dataplane-operator-index:${OPENSTACK_K8S_TAG}
-DATAPLANE_REPO                                   ?= https://github.com/openstack-k8s-operators/dataplane-operator.git
-DATAPLANE_BRANCH                                 ?= ${OPENSTACK_K8S_BRANCH}
-DATAPLANE_COMMIT_HASH                            ?=
```
Comment (gibizer) [inline Makefile:365]: There are couple of hits in ci-framework to these env vars. I'm not sure if that is active framework code or just some ci-framework self testing code https://github.com/search?q=repo%3Aopenstack-k8s-operators%2Fci-framework%20DATAPLANE_IMG&type=code

File: Makefile [hunk 2]
```diff
@@ -2123,13 +2115,13 @@ baremetal_cleanup: ## deletes the operator, but does not cleanup the service res
 
 ##@ DATAPLANE
 .PHONY: dataplane_prep
```
Comment (rabi) [inline Makefile:2117]: We probably don't need these  targets as dataplane kuttl tests already use `openstack` target.

**Comment by rabi** [comment]
recheck

**Review by rabi — COMMENTED** [review]
/lgtm

**Pull request merged**